### DISCUSSION TITLE
建议把右上角地址栏后面插件的图片，增加类似悬浮球的功能：缺省是点击开关翻译，右键进入设置。毕竟切换开关是高频操作，配置是极少使用。使用悬浮球容易遮挡内容。

### DISCUSSION THREAD

**Discussion by yuedajiong** [immersive-translate/immersive-translate#2095] [Ideas]
### 新功能 | New Feature

建议把右上角地址栏后面插件的图片，增加类似悬浮球的功能：缺省是点击开关翻译，右键进入设置。毕竟切换开关是高频操作，配置是极少使用。使用悬浮球容易遮挡内容。

### 为什么您想要添加这些功能 | Why you want to add these features

1.
2.
3.

**Comment by theowenyoung** [comment]
是否可以快捷键替代呢？ 默认是 Alt+A ，或者 右键翻译。

**Reply by yuedajiong** [reply]
可以是可以，但问题在于： 右键是几步操作（右键，去找菜单中翻译项目，点击，消除菜单）；快捷键是必须要键盘。   而这种翻译软件，大部分场景是纯鼠标的纯阅读的场景。可能数分钟数十分钟，都不用键盘。

个人使用场景的一点感受和想法，仅供参考。
